model/src/main/kotlin/config/PostgresConnection.kt

```diff
@@ -75,7 +75,45 @@ data class PostgresConnection(
     /**
      * The number of parallel transactions to use for the storage dispatcher.
      */
-    val parallelTransactions: Int = 5
+    val parallelTransactions: Int = 5,
+
+    /**
+     * The maximum number of milliseconds to wait for connections from the pool. For details see the
+     * [Hikari documentation](https://github.com/brettwooldridge/HikariCP#frequently-used).
+     */
+    val connectionTimeout: Long?,
```

Comment:
Should these all have `@JsonInclude(Include.NON_NULL)` and `null` assigned as the default value, similar to the nullable values above?